.prettierrc.yml

```diff
@@ -1,3 +1,3 @@
-trailingComma: "es5"
+trailingComma: "none"
```

Comment:
You don't like trailing commas?! I've loved them for a while for how easy it is to reorder properties/array items or comment stuff out, or add items.

It's one of the first of the first-world problems; not losing sleep over it.

Reply:
I can see that. There is just something about that dangling, unnecessary comma that really bothers me, especially when the only argument for including it is just a matter of occasional convenience. Maybe, I could be converted, but at this point, not a fan of trailing commas.